.github/workflows/pre-release.yml

```diff
@@ -0,0 +1,27 @@
+name: Pre-release
+
+permissions: read-all
+
+on:
+  push:
+    branches:
+      - release-please-*
+
+jobs:
+  pre-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        with:
+          fetch-depth: 2
+
+      - name: Install MCP Publisher
+        run: |
+          export TAG=$(curl -sL https://api.github.com/repos/modelcontextprotocol/registry/releases/latest | jq -r ".tag_name")
+          export VERSION="${TAG#v}"
```

Comment:
Locally I can verify this work, and strips the prefixed `v` but GitHub actions may differ.